src/workflowOptions.ts

```diff
@@ -63,19 +63,18 @@ export const workflowOptions = {
     name: "network",
     message: "Choose your network",
     choices(prev) {
-      if (
-        prev.path === "boilerplate-template" ||
-        prev.path === "token-staking-dapp-template"
```

Comment:
we can't support devnet for the token minter template because Irys does not support Aptos devnet yet and so we can't upload a file